aten/src/ATen/common_with_cwrap.py

```diff
@@ -50,11 +51,16 @@ def set_declaration_defaults(declaration):
         declaration['unqual_operator_name_with_overload'] = ''
     # Simulate multiple dispatch, even if it's not necessary
     if 'options' not in declaration:
-        declaration['options'] = [{'arguments': declaration['arguments']}]
+        declaration['options'] = [{
+            'arguments': copy.deepcopy(declaration['arguments']),
```

Comment:
Does this one need to be deep copied?

Reply:
yes. The calling code has some other references to this array and modifies it.